services/productService.js

```diff
@@ -0,0 +1,29 @@
+const productDao = require("../models/productDao.js");
+
+const allProducts = async (page, pageSize, category, sub, type, sort) => {
```

Comment:
해당 함수는 상품을 조회하는 코드를 포괄하는 함수이기 때문에 함수명이 다음과 같다면 더 좋을 것 같습니다.
 ```suggestion
const getProducts = async (page, pageSize, category, sub, type, sort) => {
```